cucumber/features/pan.feature

```diff
@@ -8,6 +8,7 @@ Background: Navigate to Pan page
 Given the app has launched
 And I am looking at the Pan tab
 
+@skip_ios_13
```

Comment:
I think we should not disable the whole scenario, let's disable just steps that doesn't work on iOS 13